### DISCUSSION TITLE
ミッション管理ツールにどのような機能があればいいと思いますか？

### DISCUSSION THREAD

**Discussion by sinproject-iwasaki** [sinProject-Inc/mission#10] [Ideas]
ミッション管理ツールにどのような機能があればいいと思いますか？

**Comment by sinproject-iwasaki** [comment]
- ログイン
- ミッション登録
- 難易度設定
- 納期登録、希望金額登録
- 工数集計
- 請求額計算

**Comment by REI-MORI** [comment]
現在割引○円
あと何時間で割引○円

**Comment by sinproject-iwasaki** [comment]
ゲーム的な要素

- 実積
- ログインボーナス
- デイリーミッション
- ウイークリーミッション
- 期間限定ミッション
- レベル
- 実積ポイント

試験により、グレードを設けたい。
